projects/storefrontapp/server.ts

```diff
@@ -65,7 +65,7 @@ export function app(): express.Express {
     });
   });
 
-  server.use(defaultServerErrorResponseHandlers(indexHtmlContent));
+  server.use(defaultErrorResponseHandlers(indexHtmlContent));
```

Comment:
[MEDIUM]

I can see we renamed the phrase `serverErrorResponse` to `errorResponse` in many places in this PR. But my feeling is we should have renamed it to `serverError` - just drop the `response`.
Why: we don't forward a response. We forward a raw error from Angular to Express. And we handle a raw error in ExpressJS. Because it's an error that we forward to the ExpressJS _Server_ (or Fastify Server or whatever custom http Server tehcnology for incoming traffic you might want to use) , therefore we could prefix it with `server` --> `serverError`.

What do you think about it? 

It's debatable whether `serverError` is enough descriptive - it could be potentially confused with the error in the SsrOptimizedEngine :/ . But it's hard for me to connect with the term `response` anymore - for the obejct the we pass across layers and that we handle in ExpressJS.

Reply:
Yeah... I had difficulty considering the proper names for parts of the code when we decided to drop our custom server error responses. `serverError` might be confusing (probably every error-related name will be confusing at some point). I also thought about `serverCapturedError`.
Or... What about keeping it simple as an `error`?
TBH I can't decide. We can brainstorm it together.